app/scripts/controllers/commandLine.js

```diff
@@ -15,6 +15,9 @@ angular.module('openshiftConsole')
     $scope.loginBaseURL = DataService.openshiftAPIBaseUrl();
     $scope.sessionToken = AuthService.UserStore().getToken();
     $scope.showSessionToken = false;
+    if (window.OPENSHIFT_EXTENSION_PROPERTIES && window.OPENSHIFT_EXTENSION_PROPERTIES.registry_url) {
```

Comment:
OPENSHIFT_EXTENSION_PROPERTIES should never be used by code in origin, it should only be used by code in custom extensions.  If you want to make this settable via extension JS then it should be defined (and in this case set to null) in constants.js underneath OPENSHIFT_CONSTANTS.  Constants are already available in this controller via the Constants service.

Also you don't need the `if` at all if you are using Constants

```
$scope.registryURL = Constants.REGISTRY_URL;
```

However we were still hoping to get a supported long term way to know the registry URL without requiring an extension to set it.